src/Containers/KitchenCard/KitchenCard.tsx

```diff
@@ -48,6 +46,13 @@ export const KitchenCard: React.FC<KitchenProps> = ({
     TimeComponent,
     StatusModifierComponent,
 }): React.ReactElement => {
+    const statusFormat = useCallback(
+        (orderStatus: string) => {
```

Comment:
this doesn't need any paramteres becuase it can access status from function scope.